src/components/SubscribeSection/styles.module.css

```diff
@@ -52,6 +54,19 @@
   text-align: center;
 }
 
+.rssLink {
```

Comment:
Do we want the link to stand out this much? I feel like it takes away from the subscription input box... Maybe we could make the link smaller?

Reply:
Good point, it does stand out quite a bit.